client/js/constants.js

```diff
@@ -27,10 +27,13 @@ const timeFormats = {
 	msgWithSeconds: "HH:mm:ss",
 };
 
+const sizeUnits = ["B", "kB", "MB", "GB", "TB"];
```

Comment:
```suggestion
const sizeUnits = ["B", "KiB", "MiB", "GiB", "TiB"];
```

Since it's being divided by 1024, let's use actual mebibyte units to prevent any potential confusion.